src/pages/AdminDashboard/CreateRoomForm.tsx

```diff
@@ -0,0 +1,265 @@
+import React from 'react'
+import Typography from '@material-ui/core/Typography';
+import Box from '@material-ui/core/Box';
+import { TextField } from 'formik-material-ui';
+import MuiTextField from '@material-ui/core/TextField';
+import {Autocomplete, AutocompleteRenderInputParams} from 'formik-material-ui-lab';
+import { spacing } from "@material-ui/system";
+import { styled } from "@material-ui/core/styles";
+import MuiButton from "@material-ui/core/Button";
+import { Formik, Form, Field } from 'formik';
+import CheckBoxIcon from '@material-ui/icons/CheckBox';
+import {MuiPickersUtilsProvider} from '@material-ui/pickers';
+import DateFnsUtils from '@date-io/date-fns';
+import frLocale from "date-fns/locale/fr";
+import {DateTimePicker} from 'formik-material-ui-pickers';
+import Chip from '@material-ui/core/Chip';  
+import Divider from '@material-ui/core/Divider';
+import Grid from '@material-ui/core/Grid';
+import { makeStyles, Theme, createStyles, withStyles } from '@material-ui/core/styles';
+import Flags from 'country-flag-icons/react/3x2'
+import {parsePhoneNumber} from 'libphonenumber-js'
+
+
+
+
+const Button = styled(MuiButton)(spacing);
+
+const CreateRoomForm = () => {
+
+    interface Values {
+        roomName: string;
+        participants: Array<any>;
+        startAt : Date;
+    }
+
+    const [value, setValue] = React.useState([]);
+
+
+    const ErrorChip = withStyles({
+        root: {
+
+            borderColor: "red"
+          
+        }
+      })(Chip);
+
+    const useStyles = makeStyles((theme: Theme) =>
+        createStyles({
+            chip : {
+                borderColor : "green"
+            },
+            chipIconContainer: {
+                width: 'fit-content'
+            },
+            flagIcon:{
+                height:'0.8rem',
+                marginRight:theme.spacing(1),
+                marginLeft:theme.spacing(1)
+            }
+        }),    
+    );
+
+    const classes = useStyles();
+
+    const handleKeyDown = (event) => {
+        switch (event.keyCode) {
+          case 186: // ;
+          case 32:  // space
+          case 9:{  // tab
+            event.preventDefault();
+            event.stopPropagation();
+            if (event.target.value.length > 0) {
+                const newValue = [...value, event.target.value] as never[]
+                setValue(newValue);
+            }
+            break;
+          }
+          default:
+        }
+      };
+
+    const  validateEmail = (email) =>{
```

Comment:
Could be treated in an another pr
`validateEmail`, `validatePhoneNumber`, `isParticipantValid` could be extract into a lib/helper